test/general/header_order_ranges_0.pass.cpp

```diff
@@ -27,7 +27,7 @@ main()
 {
 #if _ENABLE_RANGES_TESTING
     using namespace oneapi::dpl::experimental::ranges;
-    all_of(TestUtils::default_dpcpp_policy, views::fill(-1, 10), [](auto i) { return i == -1;});
+    all_of(TestUtils::dpcpp_policy(), views::fill(-1, 10), [](auto i) { return i == -1;});
```

Comment:
There is no `dpcpp_policy()` function in `TestUtils`.


A more general question: why do you consider `TestUtils::default_dpcpp_policy` as an improper way to e.g. create `sycl::queue` for the tests?

Reply:
> Got it - it makes sense. Could you add that into the description? I do not think it is a bug, though.

it's a bug because it doesn't terminate test process in case of exceptions like we did in `async_handler ` :
```C++
auto async_handler = [](sycl::exception_list ex_list) {
    for (auto& ex : ex_list)
    {
        try
        {
            ::std::rethrow_exception(ex);
        }
        catch (sycl::exception& ex)
        {
            ::std::cerr << ex.what() << ::std::endl;
            ::std::exit(EXIT_FAILURE);
        }
    }
};
```

Reply:
What do you think about replacing the remaining `TestUtils::default_dpcpp_policy` in the file below? 

https://github.com/uxlfoundation/oneDPL/blob/5273d6e3be5adb68e83da9cd14a0f825cc11b7d2/test/parallel_api/ranges/std_ranges_test.h#L65C95-L65C115

And, we still have these default policies, so they should be tested. I vote for doing it in `test_policies.pass`  - that should be enough. We can make `TestUtils::default_dpcpp_policy` local to that file.

Reply:
Fixed: `TestUtils::default_dpcpp_policy` - removed.

Reply:
>>> https://github.com/uxlfoundation/oneDPL/blob/5273d6e3be5adb68e83da9cd14a0f825cc11b7d2/test/parallel_api/ranges/std_ranges_test.h#L65C95-L65C115
- this function has been modified and moved into `TestUtils` namespace.

Reply:
> it's a bug because it doesn't terminate test process in case of exceptions

Thanks for the clarification, that's what I missed.